### DISCUSSION TITLE
Onedrive Personal Bidirectional Sync

### DISCUSSION THREAD

**Discussion by fivemufc** [remotely-save/remotely-save#60] [Q&A]
Hi,
Just wanted to check if this works. 
I have an iPhone and I use iCloud Drive to Sync. However I use this plugin to sync to Onedrive Personal. I wanted to check if I point my Obsidian on my PC to the remote safe folder in Onedrive --> Apps,
Will my changes made on Windows flow back to iCloud?

**Comment by fyears** [comment]
>  if I point my Obsidian on my PC to the remote safe folder in Onedrive --> Apps

You do not need to “point to Apps folder”. You just need to ensure the vault name being the same, and auth onedrive using the plugin, then the plugin would automatically use the onedrive /Apps/<vaultName> folder.

> Will my changes made on Windows flow back to iCloud?

I don’t know because I do not use iCloud. This plugin does not detect anything about iCloud, and does not have any iCloud-specific logic. It just reads local disk content. All the goal the plugin trys to achieve, is syncing the content between ondrive and local device (iphone, pc, etc). I am afraid you have to check it out by yourself.


By the way, since you are already using icloud, you could consider using the official icloud client on Windows pc.
